test/jdk/java/util/Collections/CalculateHashMapCapacityTestJMH.java

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
I don't think we need this benchmark in this PR.

Note, for future reference, benchmarks are located in the `test/micro` directory and not with the main regression suite in `test/jdk`.

Reply:
> I don't think we need this benchmark in this PR.

@stuart-marks OK, that benchmark removed.